modules/material-management/src/cryptographic_material.ts

```diff
@@ -82,14 +82,19 @@ const timingSafeEqual: (a: Uint8Array, b: Uint8Array) => boolean = (function ()
   /* https://codahale.com/a-lesson-in-timing-attacks/ */
   function portableTimingSafeEqual (a: Uint8Array, b: Uint8Array) {
     /* It is *possible* that a runtime could optimize this constant time function.
-      * Adding `eval` should prevent the optimization, but this is no grantee.
-      * If you copy this function for your own use, make sure to educate yourself.
-      * Side channel attacks are pernicious and subtle.
-      */
-    eval('') // eslint-disable-line no-eval
+     * Adding `eval` could prevent the optimization, but this is no grantee.
+     * The eval below is commented out,
+     * because if a browser is using a Content Security Policy with `'unsafe-eval'`
+     * it would fail on this eval.
+     * The value in attempting to ensure this function is not optimized,
+     * is not worth the cost of making customers to alow `'unsafe-eval'`.
+     * If you copy this function for your own use, make sure to educate yourself.
```

Comment:
Do we have a reference for such a person to read up on? Just saying "educate yourself" comes across as dismissive, and I know that was not your intent.

Reply:
This is a good point.  https://codahale.com/a-lesson-in-timing-attacks/ is linked above.  Perhaps adding `please`?  I was hoping that next sentence might mitigate some of this.

Reply:
Maybe "please review the above link before use"?